src/plugins/auto/auto_executable_network.cpp

```diff
@@ -49,16 +53,25 @@ IE::Parameter AutoExecutableNetwork::GetMetric(const std::string& name) const {
             ov::PropertyName{ov::execution_devices.name(), ov::PropertyMutability::RO}};
     } else if (name == ov::hint::performance_mode) {
         auto value = _autoSContext->_performanceHint;
-        if (!_autoSContext->_core->isNewAPI())
+        if (!_autoSContext->_core->isNewAPI()) {
+            if (_autoSchedule->_pCTPUTLoadContext) {
+                value = InferenceEngine::PluginConfigParams::THROUGHPUT;
+            }
             return value;
-        if (value == InferenceEngine::PluginConfigParams::THROUGHPUT)
+        }
+        if (value == InferenceEngine::PluginConfigParams::THROUGHPUT) {
             return ov::hint::PerformanceMode::THROUGHPUT;
-        else if (value == InferenceEngine::PluginConfigParams::LATENCY)
+        } else if (value == InferenceEngine::PluginConfigParams::LATENCY) {
             return ov::hint::PerformanceMode::LATENCY;
-        else if (value == InferenceEngine::PluginConfigParams::CUMULATIVE_THROUGHPUT)
-            return ov::hint::PerformanceMode::CUMULATIVE_THROUGHPUT;
-        else
+        } else if (value == InferenceEngine::PluginConfigParams::CUMULATIVE_THROUGHPUT) {
+            if (_autoSchedule->_pCTPUTLoadContext) {
+                return ov::hint::PerformanceMode::THROUGHPUT;
+            } else {
+                return ov::hint::PerformanceMode::CUMULATIVE_THROUGHPUT;
+            }
+        } else {
             return ov::hint::PerformanceMode::UNDEFINED;
+        }
```

Comment:
please optimize all these, looks too redundant. also, no matter newAPI or not, the perfmode property can work with string<->enum well.